Rectangular Triangle/src/main/java/org/Marta/Triangle.java

```diff
@@ -0,0 +1,10 @@
+package org.Marta;
+
+import java.util.Arrays;
+
+public class Triangle {
+    public static boolean isRectangular(int[] sides) {
+        Arrays.sort(sides);
```

Comment:
fajne rozwiązanie :) Ale proponuję, aby ta metoda zamiast otrzymywać tablicę to aby dostawała 3 boki ( sideA, sideB, sideC).

Wtedy możesz już w środku tej metody sobie stworzyć tablice i je posortować tak jak to robisz.
Aktualnie, jak masz jako argument tablicę int[] sides to nic nie stoi  na przeszkodzie, abym Ci tam wepchał:

[1,2,3] (poprawne 3 boki)
[3,4,5,6,7,8] (za dużo boków)
[] (za mało boków)